### ISSUE TITLE
Improvements for screen reading software for visually impaired users

### ISSUE THREAD

**Issue by andreanini** [andreanini/multidimensionalanalysistagger#2]
A visually impaired user reported that MAT is not compliant with screen reading software such as JAWS or NVDA. This issue should be identified and fixed.
